pandas/tests/io/json/test_normalize.py

```diff
@@ -457,6 +457,69 @@ def test_max_level_with_records_path(self, max_level, expected):
         expected_df = DataFrame(data=expected, columns=result.columns.values)
         tm.assert_equal(expected_df, result)
 
+    @pytest.mark.parametrize(
```

Comment:
To make this simpler can you just do one test which parametrizes `"Image", ["Image"], lambda x: x == "Image"`? The expectation should be the same in all cases so can move to the test body

Testing exclusion via a callable can be done as a separate test